service/controller/app/v1/key/key.go

```diff
@@ -70,8 +74,8 @@ func ToChart(v interface{}) (v1alpha1.Chart, error) {
 	return *customResource, nil
 }
 
-func Version(customResource v1alpha1.App) string {
-	return customResource.Spec.Version
+func Release(customResource v1alpha1.App) string {
+	return customResource.Spec.Release
```

Comment:
Something has gone wrong here. We renamed this field from Release to Version because the name was clearer. Can you run `dep ensure -update` again?

https://github.com/giantswarm/apiextensions/blob/1f1e998db676f6bfc5698286fd697f5b4c0a55e5/pkg/apis/application/v1alpha1/app_types.go#L121